packages/ez-react/src/components/Map.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  AnimationOptions,
+  ChartPadding,
+  Dimensions,
+  RawData,
+} from 'eazychart-core/src/types';
+import { MapPath } from './shapes/MapPath';
+import { dimensions } from 'eazychart-dev/storybook/data';
+
+export type MapChartProps = {
+  padding?: Partial<ChartPadding>;
+  dimensions?: Partial<Dimensions>;
+  animationOptions?: AnimationOptions;
+  stroke: string;
+  fill: string;
+  scale: number;
+  rawData: RawData;
+  isWrapped?: boolean;
+  projectionType: string;
+  mapData: any;
```

Comment:
```suggestion
  mapData: any;
```
Let's avoid the use of any, use '@types/geojson' ?